lib/appsignal/config.rb

```diff
@@ -189,10 +197,35 @@ def validate
         @valid = false
         @logger.error "Push api key not set after loading config"
       end
+
+      check_if_request_headers_option_is_set
     end
 
     private
 
+    def check_if_request_headers_option_is_set
+      return unless valid?
+      return if config_hash.key?(:request_headers) || env == "test"
+
+      multi_line_request_headers = SUGGESTED_REQUEST_HEADERS.map do |row|
+        row.map(&:inspect).join(", ")
+      end.join(",\n    ")
+      single_line_request_headers = SUGGESTED_REQUEST_HEADERS.flatten.join(",")
+
+      logger.warn "The `request_headers` config option was not set " \
```

Comment:
@jeffkreeftmeijer wants to log this to STDOUT, so we're going to change it. This will make it stand out more.